service/awsconfig/v6/version_bundle.go

```diff
@@ -11,8 +11,8 @@ func VersionBundle() versionbundle.Bundle {
 		Changelogs: []versionbundle.Changelog{
 			{
 				Component:   "aws-operator",
-				Description: "There are no changes yet, replace this with something useful.",
-				Kind:        versionbundle.KindChanged,
+				Description: "Fix problem getting tags data for ELBs.",
```

Comment:
Nitpick. Not sure if we could be a bit more explicit here.

Reply:
Good point. I've updated the changelog to `Get ELB tags in batches due to API restriction`.